tests/pytests/isolated/test_isolated.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
```

Comment:
`tests/pytests/isolated` directory is useless, please, move `test_isolated.py` into `tests/pytests/`.